pkg/agent/route/route_windows.go

```diff
@@ -572,3 +572,11 @@ func generateNeigh(ip net.IP, linkIndex int) *util.Neighbor {
 		State:            "Permanent",
 	}
 }
+
+func (c *Client) AddRouteForLink(dstCIDR *net.IPNet, linkIndex int) error {
+	return nil
```

Comment:
```suggestion
	return errors.New("AddRouteForLink is not implemented on Windows")
```
To avoid it being used on Windows one day without noticing there is no implementation.

Reply:
Done.